syft/frameworks/torch/fl/dataset.py

```diff
@@ -22,8 +24,19 @@ class BaseDataset:
 
     """
 
-    def __init__(self, data, targets, transform=None):
-
+    def __init__(
+        self,
+        data,
+        targets,
+        transform=None,
+        id: int = None,
+        owner=None,
+        tags: List[str] = None,
+        description: str = None,
+        child=None,
```

Comment:
for conciseness just put `**kwargs` and report them in `super().__init__(**kwargs)`

Reply:
done